deploy.php

```diff
@@ -14,6 +14,7 @@
 require __DIR__ . '/deploy/tasks/rsync.php';
 require __DIR__ . '/deploy/tasks/deploy_log.php';
 require __DIR__ . '/deploy/tasks/sentry.php';
++require __DIR__ . '/deploy/tasks/sentry_notify.php';
```

Comment:
You have an extra `+` here.

Reply:
Fixed in https://github.com/netgen/media-site/pull/90/commits/e99d654124911bf2a2371280a48559b302112e53